src/geo/data-providers/geojson/geojson-data-provider.js

```diff
@@ -82,6 +84,37 @@ GeoJSONDataProvider.prototype.generateDataForDataview = function (dataview, feat
   return data;
 };
 
+GeoJSONDataProvider.prototype.applyFilter = function (columnName, filter) {
+  var filterType;
+  var filterOptions;
+  if (filter instanceof CategoryFilter) {
+    if (filter.isEmpty()) {
+      filterType = 'accept';
+      filterOptions = { column: columnName, values: 'all' };
+    } else if (filter.get('rejectAll')) {
+      filterType = 'reject';
+      filterOptions = { column: columnName, values: 'all' };
+    } else if (filter.acceptedCategories.size()) {
+      filterType = 'accept';
+      filterOptions = { column: columnName, values: filter.getAcceptedCategoryNames() };
+    } else if (filter.rejectedCategories.size()) {
+      filterType = 'reject';
+      filterOptions = { column: columnName, values: filter.getRejectedCategoryNames() };
+    }
+  } else if (filter instanceof RangeFilter) {
+    filterType = 'range';
+    if (filter.isEmpty()) {
+      filterOptions = { column: 'age', min: 0, max: Infinity };
```

Comment:
Great catch! Thanks @fdansv!

Tests were green because I'm using the same "age" column :expressionless: